### ISSUE TITLE
Does not work on Webstorm 2016.2.4

### ISSUE THREAD

**Issue by elodszopos** [idok/sass-lint-plugin#15]
Without a `.yml` file the default inspection errors show up in the editor.
With a `.yml` file regardless of letting the plugin look for it by itself or specifically configuring which `.yml` file to use, it doesn't give any inspection errors.

Changing node interpreter version (5.11, 6.0, 6.2, 6.5) doesn't help either.

My config file, for reference:
`/project-root/.sass-lint.yml`

```
files:
  include: './src/**/*.scss'
options:
  formatter: stylish
  merge-default-rules: false
rules:
  bem-depth:
    - 0
    - max-depth: 1
  border-zero:
    - 2
    - convention: zero
  brace-style:
    - 2
    - allow-single-line: true
  class-name-format:
    - 2
    - convention: camelcase
  clean-import-paths:
    - 2
    - filename-extension: false
      leading-underscore: false
  empty-args: 2
  empty-line-between-blocks:
    - 2
    - ignore-single-line-rulesets: true
  extends-before-declarations: 2
  extends-before-mixins: 2
  final-newline:
    - 2
    - include: true
  force-attribute-nesting: 2
  force-element-nesting: 2
  force-pseudo-nesting: 2
  function-name-format:
    - 2
    - allow-leading-underscore: false
      convention: camelcase
  hex-length:
    - 2
    - style: long
  hex-notation:
    - 2
    - style: uppercase
  id-name-format:
    - 2
    - convention: camelcase
  indentation:
    - 2
    - size: 2
  leading-zero:
    - 2
    - include: false
  mixin-name-format:
    - 2
    - convention: camelcase
    - allow-leading-underscore: false
  mixins-before-declarations: 2
  nesting-depth:
    - 2
    - max-depth: 2
  no-color-keywords: 2
  no-color-literals: 2
  no-css-comments: 1
  no-debug: 2
  no-duplicate-properties: 2
  no-empty-rulesets: 2
  no-extends: 0
  no-ids: 2
  no-important: 2
  no-invalid-hex: 2
  no-mergeable-selectors: 2
  no-misspelled-properties:
    - 1
    - extra-properties: []
  no-qualifying-elements:
    - 1
    - allow-element-with-attribute: false
      allow-element-with-class: false
      allow-element-with-id: false
  no-trailing-whitespace: 2
  no-trailing-zero: 2
  no-transition-all: 2
  no-url-protocols: 2
  no-vendor-prefixes:
    - 2
    - additional-identifiers: []
      excluded-identifiers: []
  placeholder-in-extend: 2
  placeholder-name-format:
    - 2
    - convention: camelcase
  property-sort-order:
    - 2
    - ignore-custom-properties: false
  property-units:
    - 2
    - global:
        - ch
        - em
        - ex
        - rem
        - cm
        - in
        - mm
        - pc
        - pt
        - px
        - q
        - vh
        - vw
        - vmin
        - vmax
        - deg
        - grad
        - rad
        - turn
        - ms
        - s
        - Hz
        - kHz
        - dpi
        - dpcm
        - dppx
        - '%'
      per-property: {}
  pseudo-element: 2
  quotes:
    - 2
    - style: single
  shorthand-values:
    - 2
    - allowed-shorthands:
        - 1
        - 2
        - 3
  single-line-per-selector: 2
  space-after-bang:
    - 2
    - include: false
  space-after-colon:
    - 2
    - include: true
  space-after-comma:
    - 2
    - include: true
  space-before-bang:
    - 2
    - include: true
  space-before-brace:
    - 2
    - include: true
  space-before-colon: 2
  space-between-parens:
    - 2
    - include: false
  trailing-semicolon: 2
  url-quotes: 2
  variable-for-property:
    - 2
    - properties: []
  variable-name-format:
    - 2
    - allow-leading-underscore: false
      convention: camelcase
  zero-unit: 2
```

**Comment by mdrobny** [comment]
I exactly the same problem using PhpStorm:
```
PhpStorm 2016.2.2
JRE: 1.8.0_112-release-287-b3 x86_64
```

**Comment by scic** [comment]
Seems to be a duplicate of https://github.com/idok/sass-lint-plugin/issues/12

**Comment by NitroBAY** [comment]
@idok no error is reported even though CLI complains about some. The package doesn't work at all.

**Comment by Daweo93** [comment]
@idok When the update will be here?
https://plugins.jetbrains.com/plugin/8171-sass-lint

**Comment by NitroBAY** [comment]
Status is now "Wait for approval". So I guess we gotta wait jetbrains to approve it.

**Comment by Daweo93** [comment]
Hello there,
It's still don't working for me. 
I think reason is that it's in inspections for JS not for SASS.
![image](https://cloud.githubusercontent.com/assets/16107111/24355211/56ebc9f8-12f5-11e7-831b-38b7a0604648.png)
